tests/components/unifi/test_init.py

```diff
@@ -75,3 +78,30 @@ async def test_unload_entry(hass, aioclient_mock):
 
     assert await unifi.async_unload_entry(hass, config_entry)
     assert not hass.data[UNIFI_DOMAIN]
+
+
+async def test_wireless_clients(hass, hass_storage):
+    """Verify wireless clients class."""
+    entry = MockConfigEntry(domain=UNIFI_DOMAIN, data=ENTRY_CONFIG, entry_id=1)
+    hass_storage[unifi.STORAGE_KEY] = {
+        "version": unifi.STORAGE_VERSION,
+        "data": {entry.entry_id: {"wireless_devices": ["mac1", "mac2", "mac3"]}},
+    }
+
+    wireless_clients = UnifiWirelessClients(hass)
```

Comment:
Set up the integration and let it create this instance instead. We can assert the state of the storage.

Reply:
Finally fixed 😂